relay-server/config/config.go

```diff
@@ -0,0 +1,34 @@
+package config
+
+import (
+	"flag"
+
+	kg "github.com/kubearmor/KubeArmor/KubeArmor/log"
+)
+
+type KubeArmorRelayConfig struct {
+	// common config
+	GRPCPort string
+
+	// non-k8s config
+	K8s bool
```

Comment:
I think rather than having k8s flag supplied from user, we should automatically find out if the relay is executed in k8s env.

Secondly, whether to enable PushLogService should be kept configurable regardless of k8s env.